core/modules/file.py

```diff
@@ -239,6 +239,35 @@ class File(Tree):
 
 	blobCacheTime = 60 * 60 * 24  # Requests to file/download will be served with cache-control: public, max-age=blobCacheTime if set
 
+	def write(self, filename: str, content: Any, mimetype: str = "text/plain"):
+		"""
+		Write a file from any buffer into the file module.
+
+		:param filename: Filename to be written.
+		:param content:  The file content to be written, as bytes-like object.
+		:param mimetype: The file's mimetype.
+
+		:return: Returns the key of the file object written. This can be associated e.g. with a fileBone.
+		"""
+		dl_key = utils.generateRandomString()
+
+		blob = bucket.blob("%s/source/%s" % (dl_key, filename))
+		blob.upload_from_file(BytesIO(content), content_type=mimetype)
+
+		skel = self.addSkel("leaf")
+		skel["name"] = filename
+		skel["size"] = blob.size
+		skel["mimetype"] = mimetype
+		skel["dlkey"] = dl_key
+		skel["parentdir"] = None
+		skel["parententry"] = None
+		skel["pending"] = False
```

Comment:
These are already the `defaultValue`s and can be omitted to save the setitem-calls.
```suggestion
```

Reply:
You're right, but there are plenty of other locations where this is done likely. I think this has to be done in a separate pull request.